src/pykeen/models/unimodal/rgcn.py

```diff
@@ -24,15 +24,71 @@
 class RGCN(
     ERModel[torch.FloatTensor, RelationRepresentation, torch.FloatTensor],
 ):
-    """An implementation of R-GCN from [schlichtkrull2018]_.
+    r"""An implementation of R-GCN from [schlichtkrull2018]_.
 
-    This model uses graph convolutions with relation-specific weights.
+    The Relational Graph Convolutional Network (R-GCN) comprises two parts:
+
+    1. A GCN-based entity-encoder that computes enriched representations for entities, cf. `RGCNRepresentations`. 
```

Comment:
@cthoyt here we need a correct Sphinx-link to the class.

Reply:
Done